pims/tests/test_display.py

```diff
@@ -110,12 +110,19 @@ class ExportCommon(object):
     def setUp(self):
         self.frame0 = np.load(os.path.join(path, 'bulk-water_frame0.npy'))
         self.frame1 = np.load(os.path.join(path, 'bulk-water_frame1.npy'))
-        self.expected_shape = (128, 128, 3)
+        h, w = 128, 128
+        self.expected_shape = (h, w, 3)
+        self.expected_shape_rgba = (h, w, 4)
         self.expected_len = 20
 
-        self.sequence = np.random.randint(0, 255, size=(self.expected_len,) +
-                                                       self.expected_shape,
-                                          dtype=np.uint8)
+        self.sequence = np.random.randint(0, 255,
+                                          size=(self.expected_len,) +
+                                               self.expected_shape,
+                                          ).astype(np.uint8)
```

Comment:
Why remove the `dtype` kwarg and use `astype`? Isn't the original version more efficient?

Reply:
One of my Python environments has an old version of numpy that doesn't support that kwarg. I'd be OK with reverting this if you'd prefer that we modernize.